### ISSUE TITLE
Textile Support (post-Jekyll 3.0)

### ISSUE THREAD

**Issue by chrisfinazzo** [jekyll/jekyll#3654]
As this is now done through a plugin, can these references be removed? I don't see much of anything which informs people that Textile support will necessitate using the [converter][].

 If so, I will submit a PR.

(See #3319 and #3528)

[converter]: https://github.com/jekyll/jekyll-textile-converter

**Comment by parkr** [comment]
What do you mean by "these references"

**Comment by chrisfinazzo** [comment]
By this I mean, in `_docs`, within the gem, or both - whatever makes the most sense. I know it was removed from the tests in #3528, but it still exists in a few other locations. At least something to indicate that a different set of steps are required if you choose to author content with Textile. Obviously, breaking sites would be bad, so proceed with caution, if at all. As smart as some people are, I wouldn't expect them to know this straightaway. Do people read release notes? Maybe, but I would not always count on them catching this nuance. After all, it _is_ front and center on the homepage:

![screen shot 2015-04-13 at 8 46 47 pm](https://cloud.githubusercontent.com/assets/1534882/7128304/4b1479a4-e21e-11e4-984f-02558a009bf1.png)


Tl;dr Why are we still using `.textile` in examples when it is (a Essentially "opt-in" for 3.0 and b) Not widely used?

I remember seeing someone ask how many sites were actually using Textile as opposed to Mardown (GitHub stats perhaps?) and the number being really small (~1500)

**Comment by envygeeks** [comment]
Hmmm... do we not support it at all anymore, even by extension or plugin?

**Comment by chrisfinazzo** [comment]
#3319 factored it out into a plugin, I am simply suggesting that there may be places within the codebase which do not reflect this change (docs and a few examples).

E.g, "The use of Textile now requires a converter, if you intend to use this feature, please configure it"

This may have already been done in that PR, but I'm not completely sure, will check later to make sure that everything was covered.

**Comment by parkr** [comment]
There are a lot of sites that still use it. It's no longer default, but we still support it through a plugin: https://github.com/jekyll/jekyll-textile-converter

I'd rather keep it there on the docs. It highlights that we aren't just a Markdown conversion tool.
